src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/GraphDAO.scala

```diff
@@ -42,6 +44,10 @@ trait GraphDAO {
     workspacePipeline(db, workspaceNamespace, workspaceName).out(entityType).filter(hasProperty("_name", entityName))
   }
 
+  def methodConfigPipeline(db: Graph, workspaceNamespace: String, workspaceName: String, methodConfigNamespace: String, methodConfigName: String) = {
+    workspacePipeline(db, workspaceNamespace, workspaceName).out(MethodConfigEdgeType).filter(hasProperties(Map("namespace" -> methodConfigNamespace, "name" -> methodConfigName)))
```

Comment:
why do we need reserved properties here? There is no ability for the user to put in anything other than what is in the case class, it is not like an entity where the user can put in a map of whatever they want... so in effect all the properties are reserved.